experiment/resultstore/convert.go

```diff
@@ -124,6 +127,32 @@ func convert(project, details string, url gcs.Path, result downloadResult) (resu
 				URL:         buildLog, // ensure build-log.txt appears as the invocation log
 			},
 		},
+		Properties: []resultstore.Property{
+			{
+				Key:   "Job Name",
+				Value: jobName,
+			},
+			{
+				Key:   "Started Timestamp",
+				Value: strconv.FormatInt(started.Timestamp, 10),
+			},
+			{
+				Key:   "Node",
```

Comment:
Same, if we want this data to exist, we should probably put it in the fields already dedicated to it

https://github.com/googleapis/googleapis/blob/master/google/devtools/resultstore/v2/action.proto#L210

https://github.com/googleapis/googleapis/blob/cb79155f596e0396dd900da93872be7066f6340d/google/devtools/resultstore/v2/invocation.proto#L112

Reply:
Thanks. Removed, existing code handled this case.